common/src/main/java/net/adoptopenjdk/icedteaweb/logging/SystemOutLoggerFactory.java

```diff
@@ -11,6 +12,16 @@
  */
 class SystemOutLoggerFactory implements LoggerFactoryImpl {
 
+    private static final PrintStream out;
+
+    static {
+        final PrintStream tmp = System.out;
+        if (tmp.getClass().getSimpleName().contains("TeeOutputStream")) {
```

Comment:
really? can we check if the class implements an interface for example or is assignable by another class?

Reply:
sorry only the TeeOutputStream poses a problem. And it is not visible from here (different JAR)